### DISCUSSION TITLE
Consider enforcing time-zone-aware datetimes

### DISCUSSION THREAD

**Discussion by galenlynch** [AllenNeuralDynamics/aind-data-schema#830] [Ideas]
After just experiencing a leap day and daylight's savings, I think we should seriously consider making sure all of the metadata generated at AIND are time zone aware. Of course people wanting to do time arithmetic (e.g. how many hours between training sessions?) could just add this retroactively, it would make the interpretation of the data have fewer pitfalls if all date times had a proper [TZ identifier](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. "America/Los_Angeles") attached to it.

I don't know at what level to enforce this, but time zones are often overlooked until you stumble across a pitfall to storing naïve datetimes.

**Comment by saskiad** [comment]
Yep, this is something we're already working on:

https://github.com/AllenNeuralDynamics/aind-data-schema/issues/484
